tests/test_signed_payload.py

```diff
@@ -0,0 +1,45 @@
+import requests
+from eth_keys import keys
+import json
+import base64
+
+
+def print_payload_signature():
+    pk = keys.PrivateKey(b'\x01' * 32)
+
+    payload = {
+        "service_proposal": {
+            "id": 1,
+            "format": "service-proposal/v1",
+            "provider_id": "node1",
+        }
+    }
+
+    signature = pk.sign_msg(json.dumps(payload))
+    print json.dumps(payload)
+    print pk.public_key.to_checksum_address().lower()
+    print base64.b64encode(signature.to_bytes())
+
+
+def test_signed_payload():
```

Comment:
If we update this to `unittest`, we could use assertions and run all tests together with nice UI :)

Reply:
updated